agent/consul/leader_connect_ca.go

```diff
@@ -1451,6 +1451,22 @@ func (c *CAManager) SignCertificate(csr *x509.CertificateRequest, spiffeID conne
 		defer c.caLeafLimiter.csrConcurrencyLimiter.Release()
 	}
 
+	// Append our local CA's intermediate if there is one.
```

Comment:
This comment line should probably get moved down to 1484 (where the intermediate gets appended)